<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>629</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Config dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="envTab">
      <attribute name="title">
       <string>Environment</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QGroupBox" name="bgGroupBox">
           <property name="title">
            <string>Background</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLabel" name="backgrounSetupLabel">
                  <property name="text">
                   <string>Setup background</string>
                  </property>
                  <property name="buddy">
                   <cstring>bgPathLineEdit</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="bgPathLineEdit">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="browseBgButton">
                  <property name="text">
                   <string>Browse...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="resetBgButton">
                  <property name="text">
                   <string>Reset</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="tableBgSetupLabel">
                  <property name="text">
                   <string>Setup tableBg</string>
                  </property>
                  <property name="buddy">
                   <cstring>bgPathLineEdit</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="tableBgPathLineEdit">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <spacer name="tableBghorizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="browseTableBgButton">
                  <property name="text">
                   <string>Browse...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="resetTableBgButton">
                  <property name="text">
                   <string>Reset</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QCheckBox" name="noIndicatorCheckBox">
                <property name="text">
                 <string>No indicator</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="noEquipAnimCheckBox">
                <property name="text">
                 <string>No equip anim</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="UseLordBackdropBox">
                <property name="text">
                 <string>Use Lord Backdrop</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="audioGroupBox">
           <property name="title">
            <string>Audio</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="setupBgMusicLabel">
                <property name="text">
                 <string>Setup background music</string>
                </property>
                <property name="buddy">
                 <cstring>bgMusicPathLineEdit</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="bgMusicPathLineEdit">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="browseBgMusicButton">
                  <property name="text">
                   <string>Browse...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="resetBgMusicButton">
                  <property name="text">
                   <string>Reset</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QCheckBox" name="enableLastWordCheckBox">
                <property name="text">
                 <string>Enable last word</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QCheckBox" name="enableBgMusicCheckBox">
                <property name="text">
                 <string>Enable background music</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="enableEffectCheckBox">
                <property name="text">
                 <string>Enable effects</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QCheckBox" name="UseLordBGMBox">
                <property name="text">
                 <string>Enable lord background music</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="bgmVolumeLabel">
                <property name="text">
                 <string>BGM    Volume</string>
                </property>
                <property name="buddy">
                 <cstring>bgmVolumeSlider</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="bgmVolumeSlider">
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="effectVolumeLabel">
                <property name="text">
                 <string>Effect Volume</string>
                </property>
                <property name="buddy">
                 <cstring>bgmVolumeSlider</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="effectVolumeSlider">
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="fontSetupGroupBox">
           <property name="title">
            <string>Font setup</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="appFontLabel">
                <property name="text">
                 <string>Application font</string>
                </property>
                <property name="buddy">
                 <cstring>appFontLineEdit</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="appFontLineEdit">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="changeAppFontButton">
                <property name="text">
                 <string>Set application font</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="textEditFontLabel">
                <property name="text">
                 <string>Text edit font</string>
                </property>
                <property name="buddy">
                 <cstring>textEditFontLineEdit</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="textEditFontLineEdit">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="setTextEditFontButton">
                <property name="text">
                 <string>Font...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="setTextEditColorButton">
                <property name="text">
                 <string>Color...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gameTab">
      <attribute name="title">
       <string>Game</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QCheckBox" name="neverNullifyMyTrickCheckBox">
             <property name="text">
              <string>Never nullify my single target trick</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="autoTargetCheckBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Enable auto target</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="intellectualSelectionCheckBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Enable intellectual selection</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="doubleClickCheckBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Enable double-click</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="defaultHeroskinCheckBox">
             <property name="text">
              <string>use default hero skin</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="autoUpdateCheckBox">
             <property name="text">
              <string>Enable Auto Update</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
             <item>
              <widget class="QLabel" name="bubbleChatBoxDelayLabel">
               <property name="text">
                <string>Bubble chat box delay(seconds)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="bubbleChatBoxDelaySpinBox">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>5</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="gameRecordGroupBox">
           <property name="title">
            <string>Game Record</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_18">
              <item>
               <widget class="QCheckBox" name="enableAutoSaveCheckBox">
                <property name="text">
                 <string>Enable Auto Save</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="networkOnlyCheckBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Network Game Only</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <widget class="QLabel" name="recordPathSetupLabel">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Setup Record Paths</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="recordPathSetupLineEdit">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_1">
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="browseRecordPathButton">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Browse...</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="resetRecordPathButton">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Reset</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>browseBgMusicButton</tabstop>
  <tabstop>bgMusicPathLineEdit</tabstop>
  <tabstop>bgPathLineEdit</tabstop>
  <tabstop>resetBgMusicButton</tabstop>
  <tabstop>enableLastWordCheckBox</tabstop>
  <tabstop>bgmVolumeSlider</tabstop>
  <tabstop>effectVolumeSlider</tabstop>
  <tabstop>browseBgButton</tabstop>
  <tabstop>resetBgButton</tabstop>
  <tabstop>noIndicatorCheckBox</tabstop>
  <tabstop>noEquipAnimCheckBox</tabstop>
  <tabstop>UseLordBackdropBox</tabstop>
  <tabstop>tableBgPathLineEdit</tabstop>
  <tabstop>browseTableBgButton</tabstop>
  <tabstop>resetTableBgButton</tabstop>
  <tabstop>changeAppFontButton</tabstop>
  <tabstop>appFontLineEdit</tabstop>
  <tabstop>textEditFontLineEdit</tabstop>
  <tabstop>setTextEditFontButton</tabstop>
  <tabstop>setTextEditColorButton</tabstop>
  <tabstop>enableAutoSaveCheckBox</tabstop>
  <tabstop>recordPathSetupLineEdit</tabstop>
  <tabstop>browseRecordPathButton</tabstop>
  <tabstop>resetRecordPathButton</tabstop>
  <tabstop>neverNullifyMyTrickCheckBox</tabstop>
  <tabstop>autoTargetCheckBox</tabstop>
  <tabstop>intellectualSelectionCheckBox</tabstop>
  <tabstop>doubleClickCheckBox</tabstop>
  <tabstop>defaultHeroskinCheckBox</tabstop>
  <tabstop>autoUpdateCheckBox</tabstop>
  <tabstop>bubbleChatBoxDelaySpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
